tjpcov/covariance_cluster_counts_gaussian.py

```diff
@@ -0,0 +1,73 @@
+from .covariance_clusters import CovarianceClusters
+import numpy as np
+
+
+class ClusterCountsGaussian(CovarianceClusters):
```

Comment:
This class basically computes the Gaussian contribution to the clusters covariance, which is the shot noise. I think this should me mentioned in the docstring and the method defined here too (instead of CovarianceClusters). Or is it going to be usable in some other case?.

Reply:
I checked the N x C_ell and it doesnt use the shotnoise, so I moved it down to this class.